engine/src/main/java/org/terasology/rendering/openvrprovider/OpenVRProvider.java

```diff
@@ -0,0 +1,332 @@
+package org.terasology.rendering.openvrprovider;
+
+import com.sun.jna.Memory;
+import com.sun.jna.NativeLibrary;
+import com.sun.jna.Pointer;
+import jopenvr.*;
```

Comment:
Code style generally favors writing out imports instead of allowing wildcards. Also, the `jopenvr` package doesn't include a domain? That's sort of odd.